exonum/src/api/websocket.rs

```diff
@@ -346,18 +346,20 @@ impl Handler<Broadcast> for Server {
 impl Handler<Transaction> for Server {
     type Result = Result<TransactionResponse, failure::Error>;
 
+    /// Checks if incoming transaction is valid with Blockchain::check_tx.
+    /// Broadcasts valid transactions and ignores invalid.
```

Comment:
Nit: I'd rephrase it like "Broadcasts transaction if the check was passed, and returns an error otherwise.` (something similar can be applied to the `ExplorerApi::add_transaction` as well).